Autopilot/AttitudeManager/InputCapture.c

```diff
@@ -285,6 +296,7 @@ void __attribute__((__interrupt__, no_auto_psv)) _IC7Interrupt(void)
     } else {
         end_time[6] = IC7BUF;
         new_data_available[6] = 1;
+        last_capture_time = getTime(); //Capture the time. Used for detecting disconnects
     }
```

Comment:
As discussed, the receiver is set up so that it outputs 0 on all PWM channels on a disconnect